doc-src/ch-fromhost_desc.rst

```diff
@@ -95,27 +95,38 @@ Additional arguments
 :code:`--cray-mpi` dependencies and quirks
 ==========================================
 
-The implementation of :code:`--cray-mpi` for MPICH is messy, foul smelling,
-and brittle. It replaces or overrides the open source MPICH libraries
-installed in the container. Users should be aware of the following.
+The implementation of :code:`--cray-mpi` is messy, foul smelling, and brittle.
+It replaces or overrides the MPICH or OpenMPI libraries installed in the
+container. Users should be aware of the following.
+
+NOTE: A given item relates to both MPICH and OpenMPI unless otherwise stated.
 
 1. Containers must have the following software installed:
 
-   a. Open source `MPICH <https://www.mpich.org/>`_.
+   a. Corresponding open source MPI implementation.
+
+   b. (MPICH) `PatchELF with our patches <https://github.com/hpc/patchelf>`_.
+      Use the :code:`shrink-soname` branch.
```

Comment:
I recommend this style:

```
   b. `PatchELF with our patches <https://github.com/hpc/patchelf>`_. Use the
      :code:`shrink-soname` branch. (MPICH only.)

   c. Blah blah blah. (MPICH and OpenMPI.)
```

Reply:
My reasoning was by putting it first the item could be skipped at a glance if it wasn't relevant after is certainly cleaner. Done.